OpenMCSphericalShellFunctions/OpenMC_xml.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Aug  7 10:07:33 2024
+
+@author: Anupama Rajendra
+"""
+
+import openmc
+import argparse
+
+parser = argparse.ArgumentParser(description="Specify required inputs: file path to ALARA Element Library, element name, inner radius [cm], outer_radius [cm], particle energy [eV]")
+
+#Required user inputs:
+parser.add_argument('--filepath', type=str, required=True)    

```

Comment:
This appears to the path to the ALARA element library?  If so, it should have a better name than just `filepath`